### DISCUSSION TITLE
Answer LG question automatically

### DISCUSSION THREAD

**Discussion by malip2** [Homebrew/discussions#5272] [Getting started]
Hi, 
hope it is ok that I ask a question here. 
I have a receiver connected and the LG with homebrew wants to switch it on. 
There is a message like "do you want to power on the receiver" which I answer with yes allways manually.

Do you have an idea to automate this like to write a linux runlevel script with the keycode for ok? 
Is lginouthook maybe an idea?

Another question I have: Is there a matrix available to see if an LG firmware update version  works with the latest homebrew or is it just luck to update? 

Regards  Markus

sorry if I am wrong here.

Regards Markus
![IMG_9444](https://github.com/Homebrew/discussions/assets/31176881/a770db40-0d57-4a2b-81fb-f82ef5a0730b)

**Comment by gromgit** [comment]
You're mixing up your brews. 😀 

This Homebrew is [a package manager for macOS and Linux](https://brew.sh), nothing to do with smart TVs from any manufacturer (yet?).

I think you want [these guys instead](https://www.webosbrew.org).
